internal/server/iigointernal/history.go

```diff
@@ -0,0 +1,22 @@
+package iigointernal
+
+import "github.com/SOMAS2020/SOMAS2020/internal/common/rules"
+
+type Accountability struct {
+	clientID int
+	pairs    []rules.VariableValuePair
+}
+
+var TurnHistory []Accountability
```

Comment:
should this not be stored in `gamestate` instead? or is this required to be used?